gennav/planners/graph_search/astar.py

```diff
@@ -113,5 +114,5 @@ def astar(graph, start, end, heuristic={}):
             if flag == 1:
                 open_.append(neighbour)
     # if path doesn't exsist it returns just the start point as the path
-    path = [start]
+    path = [RobotState(position=start)]
```

Comment:
Is it posiible to make this return a `Trajectory` instead of list of `RoboStates`